src/eXpansion/Storm/ObstacleRecords/ObstacleRecordsBundle.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace eXpansion\Storm\ObstacleRecords;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+/**
+ * Class EmotesBundle
+ *
+ * @package eXpansion\Bundle\Emotes
+ */
+class ObstacleRecordsBundle extends Bundle
```

Comment:
I understand this even less. When proper dataProviders are coded records will work automatically. 

+Should't have bundles in anything else beside eXpansion/Bundles